apps/prairielearn/src/ee/cron/workspaceHostLoads.ts

```diff
@@ -114,29 +140,29 @@ const cloudwatch_definitions = {
   },
 };
 
-async function sendStatsToCloudwatch(stats: WorkspaceLoadStats) {
+async function sendStatsToCloudwatch({ timestamp_formatted, ...numericStats }: WorkspaceLoadStats) {
```

Comment:
The object destructuring helped typescript understand this better than the `key !== 'timestamp_formatted'`